AppBundles/DataCheckerPlugin/DataCheckerAutomation.cs

```diff
@@ -144,11 +178,17 @@ private void ExtractDrawingList()
                            !lowName.Contains(oldVersionMask);
                 })
                 .Select(path => path.Substring(index))
+                .OrderBy(path => System.IO.Path.GetFileName(path), new DefaultDocComparer(fileName))
                 .ToArray();
 
+            LogTrace("DEFAULT drawing is: " + (drawings.Count()>0 ? drawings[0] : null));
+
             SaveAsJson(drawings, "drawings-list.json"); // the file name must be in sync with activity definition
 
-            AddMessage($"Found {drawings.Length} drawings", Severity.Info);
+            AddMessage($"Found {drawings.Count()} drawings", Severity.Info);
```

Comment:
It's NBD for the given case, but in general `Count()` is slower comparing to `Length`:
https://stackoverflow.com/a/981283

Reply:
I have Length localy, but unable to merge now, need to resolve many rebases :-(